.github/workflows/build-run-applications.yml

```diff
@@ -165,22 +184,58 @@ jobs:
       - name: Install Python packages
         env:
           PIP_EXTRA_INDEX_URL: "https://dl.espressif.com/pypi/"
-        run: pip install --prefer-binary cryptography pytest-embedded pytest-embedded-serial-esp pytest-embedded-idf pytest-custom_exit_code
+        run: |
+          echo "PYTEST_BENCHMARK_IGNORE=--ignore='examples/display_lvgl_benchmark'" >> $GITHUB_ENV
+          pip install --prefer-binary cryptography pytest-embedded pytest-embedded-serial-esp pytest-embedded-idf pytest-custom_exit_code
+      - name: Download latest results
+        uses: actions/download-artifact@v4
+        with:
+          pattern: benchmark_*
+          path: benchmark/
+      - name: Set ignores
+        if: contains(github.event.pull_request.labels.*.name, 'Run benchmark') || contains(inputs.WFType, 'Build + Tests + Benchmark') || github.ref_name == 'master'
+        id: set_ignores
+        run: |
+          echo "PYTEST_BENCHMARK_IGNORE=" >> $GITHUB_ENV
+      - name: Pull
+        run: |
+          git pull --rebase origin ${{ github.head_ref }} || echo "No remote changes to rebase"
```

Comment:
Why is this step needed?

Reply:
Removed. I forgot remove it after some other changes and tests... Thanks